README.md

```diff
@@ -15,64 +15,6 @@ The **API docs** are generated from our [Rust](https://github.com/tauri-apps/tau
 $ yarn
```

Comment:
Kinda weird that this single command is still here ngl (well moving the command in general is kinda weird but alright). We should add a link from the readme to the contributing guide tho, for the less github savvy people.

Reply:
fixed!